### ISSUE TITLE
Restrict access based on github organizations

### ISSUE THREAD

**Issue by agriffis** [progrium/envy#46]
Unless I'm misunderstanding how things work, using envy allows anybody with a github account to run containers on my host. It would be nice to easily restrict to my github user, or even better, to a github organization.

**Comment by progrium** [comment]
That's right. Github orgs in the future, but for now (since the Go port was
merged into master): https://github.com/progrium/envy/issues/13

On Wed, Aug 12, 2015 at 6:58 PM, Aron Griffis notifications@github.com
wrote:

> Unless I'm misunderstanding how things work, using envy allows anybody
> with a github account to run containers on my host. It would be nice to
> easily restrict to my github user, or even better, to a github organization.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/progrium/envy/issues/46.

## 

Jeff Lindsay
http://progrium.com
